pkg/local_object_storage/writecache/mode.go

```diff
@@ -11,6 +11,9 @@ import (
 // ErrReadOnly is returned when Put/Write is performed in a read-only mode.
 var ErrReadOnly = logicerr.New("write-cache is in read-only mode")
 
+// ErrNotInitialized is returned when write-cache is initializing.
+var ErrNotInitialized = logicerr.New("write-cache is not initialized")
```

Comment:
I would add ` yet`, not to spread panic among testers, administrators and service engineers.